components/session-based-conditional-auth-functions/src/main/java/org/wso2/carbon/identity/extensions/authenticator/conditional/auth/functions/model/Session.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.wso2.carbon.identity.extensions.authenticator.conditional.auth.functions.model;
+
+import org.json.JSONObject;
+
+/**
+ * Model class to store details about a active sessions of user
+ */
+public class Session {
+
+    private String sessionId;
+    private String timeStamp;
+    private String userAgent;
+    private String ipAddress;
+    private String serviceProvider;
+
+    /**
+     * Constructor of the model class
```

Comment:
All the java doc comments should end with the period "." Better if we could follow that.

Reply:
period added to all method level comments and class level comments